.github/workflows/dcm.yaml

```diff
@@ -0,0 +1,38 @@
+# A CI workflow to run the Dart Code Metrics analyzer (https://dcm.dev).
+
+# NOTE: DCM usage in DevTools is currently experimental! 
+
+# The DCM_CI_KEY and DCM_EMAIL secrets are set in the admin settings page
+# of https://github.com/flutter/devtools. These are associated with a trial
+# license of DCM, and will need to be changed if we decide to purchase an
+# actual license. The trial license will expire on April 11, 2024. 
```

Comment:
how would someone who needs to set things up again get a copy of these secrets?

Reply:
@jacob314 For this, should we say reach out to you or file a bug to get access to the secrets?

Reply:
We chatted about how to handle this at DevTools standup today. Added a link to the secrets to the comment here, and added suggestions to https://github.com/flutter/devtools/pull/5629